tempesta_fw/classifier/req_conn_limit.c

```diff
@@ -37,6 +37,7 @@
 #include "../classifier.h"
```

Comment:
Please rename the file to frang.c. This module does more than just requests and connections limiting.